src/components/Requirements/RequirementGroup.vue

```diff
@@ -92,6 +92,8 @@ const reqGroupColorMap = {
   College: ['4D7D92', 'sangBlue'],
   Major: ['148481', 'emGreen'],
   Minor: ['105351', 'chrisGreen'],
+  // TODO: make colors wrap around if less than 4 elements, else new colour
+  Grad: ['105351', 'sangBlue'],
```

Comment:
If we want sangBlue should the first element be the same as that of College? So 4D7D92?

Reply:
I was going to resolve this on my end when I make sure the requirements bar front-end works! This is just a placeholder we decided on at the dev meeting haha

Reply:
Sounds good!